lsp/jsonnet_ls.lua

```diff
@@ -0,0 +1,24 @@
+--- @brief
+---
+--- https://github.com/grafana/jsonnet-language-server
+---
+--- A Language Server Protocol (LSP) server for Jsonnet.
+---
+--- The language server can be installed with `go`:
+--- ```sh
+--- go install github.com/grafana/jsonnet-language-server@latest
+--- ```
+
+local util = require 'lspconfig.util'
+
+return {
+  cmd = { 'jsonnet-language-server' },
+  filetypes = {
+    'jsonnet',
+    'libsonnet',
+  },
+  root_dir = function(fname)
+    return util.root_pattern 'jsonnetfile.json'(fname)
+      or vim.fs.dirname(vim.fs.find('.git', { path = fname, upward = true })[1])
+  end,
```

Comment:
Looks like you should be able to replace this with `root_markers`?

Reply:
I wasn't sure about that, is that as easy as my last commit?